packages/core/admin/admin/src/content-manager/components/BlocksEditor/BlocksInput/index.js

```diff
@@ -74,6 +74,35 @@ const BlocksInput = ({ disabled, placeholder }) => {
     }
   };
 
+  /**
+   * Modifier keyboard shortcuts
+   */
+  const handleKeyboardShortcuts = (event) => {
+    const isMac = /Mac|iPod|iPhone|iPad/.test(navigator.userAgent);
+    const isCtrlOrCmd = isMac ? event.metaKey : event.ctrlKey;
+
+    if (isCtrlOrCmd) {
+      switch (event.key) {
+        case 'b':
```

Comment:
I think the shortcut should be stored in the useModifiers under a new key. So that the store remains the single source of truth